app/presenters/queries/expanded_link_set.rb

```diff
@@ -3,10 +3,11 @@ module Queries
     class ExpandedLinkSet
       attr_reader :draft
 
-      def initialize(content_id:, draft: false, locale: Edition::DEFAULT_LOCALE)
+      def initialize(content_id: nil, locale: nil, edition: nil, draft: false)
```

Comment:
Ah dang - it bugs me a bit having both interfaces in as it ups the complexity and chances of inconsistency. I suppose since you can use GetExpandedLinks without an edition (via our endpoint) we can't fix that though.